versions.props

```diff
@@ -43,13 +44,17 @@ io.airlift:airline = 0.8
 io.atomix.catalyst:* = 1.1.2
 io.atomix.copycat:* = 1.2.0
 io.atomix:* = 1.0.0-rc9
-io.dropwizard.metrics:metrics-* = 3.2.3
+io.dropwizard.metrics:metrics-* = 4.1.5
```

Comment:
Will this not effectively cause a bump in version for anyone that consumes atlasdb?

Reply:
Yes, it will. I'll see if we can separate the test and compile classpaths.

Reply:
we can't be on dropwizard metrics 4

Reply:
it has enough breaking changes here

Reply:
Now we're not. I went back to metrics 3